### DISCUSSION TITLE
[pool]i can't check fee after join a pool

### DISCUSSION THREAD

**Discussion by lpf763827726** [Chia-Network/chia-blockchain#7078] [Ideas]
i can see how much fee when i join the pool
![2021-07-01_12-24](https://user-images.githubusercontent.com/43004977/124064449-62dbf100-da67-11eb-9d5f-ccbcc6b87817.png)

but i can't check anytime after join the pool
![2021-07-01_12-26](https://user-images.githubusercontent.com/43004977/124064523-8acb5480-da67-11eb-8f61-59371db113f1.png)

and when i try to change fee = 99.9%(farmer has no one TXCH) in my poolserver secretly, my chia even don't know have change(if this pool have many farmer, i will just think maybe I don't have enough points after getting a little bit of TXCH)

(now i only know i can check it in `ADD A PLOT NET` and `CHANGE POOL` button and input the pool_url, it can see in Verify Pool Details, but i think i won't check it every time again, i think i should show out in `Your Pool Overview`)

**Comment by lpf763827726** [comment]
i think if meet some pool are dishonesty is very terrible

**Comment by loppefaaret** [comment]
you seem to be on to something here - something we might have to do better in the GUI

**Reply by lpf763827726** [reply]
I think pool may also need to make some changes will be better. You can try to make some adjustments to the rules of sending money in the pool. For example, after changing the tip, it will be paid after few hours (to prevent the pool owner from making changes a few minutes before distributing XCH)

**Reply by lpf763827726** [reply]
i still have a problem, i can check it in GUI anytime, but how can i check it with out GUI? or some pool change the message prompt

**Reply by loppefaaret** [reply]
at some point, you need to establish a certain degree of trust, with the pool of your choice - if they go for a weeks earnings, instead of running a pool for 2 months, and make just as much, it is their headache, and potential legal problems.
try and be a bit catious about which pools you trust your plots and earnings with!

**Reply by lpf763827726** [reply]
yea, you are right, Maybe I'm too suspicious of the pool

(because i test pool with my friend one night, i test the plot and farmer, he test the pool, when i want to afk, let it farm by itself, my friend change the fee, and i found something wrong when i check the wallet in the next day)
